data/populationData.tsv

```diff
@@ -240,3 +240,5 @@ FRA-Martinique	364354	France	882	20	Tropical	None	None	None
 FRA-Mayotte	270372	France	300	15	Southern	None	None	None
 Israel	9136000	Israel	15871	758	Northern	None	None	None
 Iceland	364260	Iceland	1169	29	Northern	None	None	None
+JPN-Okinawa	1433566	Japan	100	24	Tropical	None	None	None
+
```

Comment:
This empty line currently breaks a script downstream, please remove it. While you are at it, can you put the website you mentioned as src for the ICU bed number (last column?)